addons/survey/models/survey_survey.py

```diff
@@ -60,6 +59,24 @@ def _get_default_access_token(self):
     answer_count = fields.Integer("Started", compute="_compute_survey_statistic")
     answer_done_count = fields.Integer("Completed", compute="_compute_survey_statistic")
 
+    # scoring and certification fields
+    scoring_type = fields.Selection([
+        ('no_scoring', 'No scoring'),
+        ('scoring_with_answers', 'Scoring with answers at the end'),
+        ('scoring_without_answers', 'Scoring without answers at the end')],
+        string="Scoring", required=True, default='no_scoring')
+    passing_score = fields.Float('Passing score (%)', required=True, default=80.0)
+    limit_number_of_attempts = fields.Boolean('Limited number of attempts',
+        help="Check this option if you want to limit the number of attempts per user")
+    allowed_attempts_count = fields.Integer('Number of attempts', default=1)
+    survey_time_limited = fields.Boolean('The survey is limited in time')
+    survey_time_limit = fields.Float("Time limit (minutes)")
```

Comment:
Why prefixing them by `survey` as we are on survey model ? 
I'd rather go for `is_time_limited` and `time_limit`.
Also, we should force the limit to be positive when 'is_limited' is check (with SQL constraint + onchange setting a default value when allowing the limit).